docs/guides/build-with-magicavoxel.md

```diff
@@ -30,22 +30,20 @@ from the App Store in the *Security & Privacy* settings:
 
 ## Tutorial
 
-A good basic introduction to using MagicaVoxel is to watch the official
-tutorial video as the author quickly runs through a lot of the features in 10
-minutes:
+The official tutorial video serves as a good introduction to using MagicaVoxel.
+The author quickly runs through many basic features in 10 minutes:
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/PPu7SJ1_bwc" frameborder="0" allowfullscreen></iframe>
 
-The best way to learn is to click around, play with the UI, and pay attention
-to the tooltips on hover. We were able to create a scene in less than 20
-minutes without having ever used the program nor watched any tutorials. But
-given the absence of official tutorials, we will provide guidance on navigating
-the UI.
+The best way to navigate the MagicaVoxel UI is to click around, play with application controls,
+and pay attention to the tooltips on hover. Many developers are able to create their first
+scene within minutes!  Given the absence of official tutorials, however,
```

Comment:
I wanted to avoid putting a number to it; puts a standard in people's head about how fast they should learn.  Maybe "within a few hours" would be better though?  Not sure.